proto/api_v2/query.proto

```diff
@@ -112,6 +115,7 @@ message TraceQueryParameters {
     (gogoproto.nullable) = false
   ];
   int32 search_depth = 8;
+  bool raw_traces = 9;
```

Comment:
Same for Find?

Reply:
@yurishkuro this is for `FindTraces`

Reply:
oh - let's add the same documentation to the field.

Also, why are you limiting this to api_v2 only? Let's make the same change to api_v3

Reply:
@yurishkuro I was going to do v3 in a separate PR but let me know if you prefer it in this one.

Reply:
https://github.com/jaegertracing/jaeger-idl/pull/115

Reply:
I would do both in one